lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -1343,4 +1343,32 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
                                                           data, dimValueList);
         return success();
       });
+
+  patterns.onOp("Tile", 13,
+                [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+                  Torch::ValueTensorType resultType;
+                  Value operand;
+                  Value repeatDims;
+                  if (binder.tensorOperands(operand, repeatDims) ||
+                      binder.tensorResultType(resultType))
+                    return failure();
+                  Torch::PrimTolistOp repeatDimsToList = rewriter.create<Torch::PrimTolistOp>(
+                  binder.getLoc(),
+                  Torch::ListType::get(rewriter.getType<Torch::IntType>()), repeatDims);
```

Comment:
I think you have to extract value from repeatDims tensor and then create a list out of it. 
you can use `torch.index_select` to extract values.

Reply:
Thanks for pointing this out, I had used torch.aten.item to extract individual elements for the 1-D tensor, will change it to index_select

Reply:
It seems there should be a better way to do this 
The earlier implementation had `PrimTolistOp` (which I think comes from [Torch.Tensor.toList](https://pytorch.org/docs/stable/generated/torch.Tensor.tolist.html#torch.Tensor.tolist) )
There is also `Aten_TensorToListOp` which would be the exact fit here but I think it is not yet lowered to linalg.

I could possibly take this up next as there is a recurring need to convert `1-D tensors` to `List(int)` for onnx to torch lowering

Reply:
I think we can assign `Aten_TensorToListOp` to any folk from the turbine camp, would be a good learning exercise for them. @kumardeepakamd, do you want to assign this to someone?

Reply:
Marking this resolved will update with `Aten_TensorToListOp` in a future PR as needed.